pkgs/tools/networking/pihole-ftl/default.nix

```diff
@@ -0,0 +1,42 @@
+{ cmake, gmp, fetchFromGitHub, libidn, nettle, readline, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "pihole-ftl";
+  version = "5.3.4";
```

Comment:
Since this hasn't been merged yet, would it be possible to bump to `5.5.1`? It looks like a vulnerability was patched in `5.5`.

Reply:
Sent out an update to 5.7